tools/accuracy_checker/sample/sample_blob_config.yml

```diff
@@ -6,10 +6,8 @@ models:
       # launcher framework (e.g. caffe, openvino)
       - framework: openvino
         # device for infer (e.g. for openvino cpu, gpu, hetero:cpu,gpu ...)
-        # Note: not all devices support blob execution, it is the reason why for sample we will use myriad.
-        device: MYRIAD
+        device: CPU
```

Comment:
I'm not sure that it will work. I can recommend remove this sample config and its mention  too